rest-service/manager_rest/test/security/test_authentication.py

```diff
@@ -32,8 +33,18 @@
 class AuthenticationTests(SecurityTestBase):
 
     def test_secured_client(self):
+        alice = self.sm.list(management_models.User,
+                             filters={'username': 'alice'})
+        self.assertEqual(len(alice), 1)
+        login_time = alice[0].last_login_at
+        self.assertEqual(login_time, None)
         self._assert_user_authorized(username='alice',
                                      password='alice_password')
+        alice = self.sm.list(management_models.User,
+                             filters={'username': 'alice'})
+        self.assertEqual(len(alice), 1)
+        login_time = alice[0].last_login_at
+        self.assertNotEqual(login_time, None)
```

Comment:
also add a unittest that does the same, but with the status endpoint, and checks that the last login didnt change?

and in the test also try passing a wrong password, which currently would 500 if the db was indeed readonly (which it won't be in the unittest)